hydra-explorer/web/src/providers/HeadsDataProvider/index.tsx

```diff
@@ -21,7 +21,7 @@ export const HeadsDataProvider: React.FC<any> = ({
   const [error, setError] = useState<string | null>(null)
 
   useDataFetcher<HeadState[]>({
-      url: '/heads',
+      url: 'http://explorer.hydra.family/heads',
```

Comment:
Must: not hard-code this in case someone is running a docker container locally with different data!